README.md

```diff
@@ -156,14 +158,23 @@ Novu provides a single API to manage providers across multiple channels with a s
 - [x] [Telnyx](https://github.com/novuhq/novu/tree/main/providers/telnyx)
 - [x] [Termii](https://github.com/novuhq/novu/tree/main/providers/termii)
 - [x] [Gupshup](https://github.com/novuhq/novu/tree/main/providers/gupshup)
+- [x] [SMS Central](https://github.com/novuhq/novu/tree/main/providers/sms-central)
+- [x] [Maqsam](https://github.com/novuhq/novu/tree/main/providers/maqsam)
+- [x] [46elks](https://github.com/novuhq/novu/tree/main/providers/forty-six-elks)
+- [x] [Clickatell](https://github.com/novuhq/novu/tree/main/providers/clickatell)
+- [x] [Burst SMS](https://github.com/novuhq/novu/tree/main/providers/burst-sms)
+- [x] [Firetext](https://github.com/novuhq/novu/tree/main/providers/firetext)
+- [x] [Infobip](https://github.com/novuhq/novu/tree/main/providers/infobip)
+- [x] [SNS](https://github.com/novuhq/novu/tree/main/providers/sns)
 - [ ] Bandwidth
```

Comment:
Same here. Any reason why we are not linking to it?

Reply:
These providers are not yet available